src/classes/Query.cls

```diff
@@ -1331,6 +1349,7 @@ public class Query {
     private Object conditionArgs19;
 
     private String conditionOperator = 'AND';
+    private String optionalClause = ' ';
```

Comment:
If I were you, I would initialize the string with empty string '', so that we can skip the check in the `formQueryString()` method.